micro-rdk-nmea/src/parse_helpers/enums.rs

```diff
@@ -0,0 +1,329 @@
+    ),
+    (
+        27,
+        PositionReportForLongRangeApplications,
+        "Position report for long range applications"
+    ),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    PositionAccuracy,
+    (0, Low, "Low"),
+    (1, High, "High"),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    RaimFlag,
+    (0, NotInUse, "not in use"),
+    (1, InUse, "in use"),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    TimeStamp,
+    (60, NotAvailable, "Not available"),
+    (61, ManualInputMode, "Manual input mode"),
+    (62, DeadReckoningMode, "Dead reckoning mode"),
+    (
+        63,
+        PositioningSystemIsInoperative,
+        "Positioning system is inoperative"
+    ),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    AisTransceiver,
+    (0, ChannelAVdlReception, "Channel A VDL reception"),
+    (1, ChannelBVdlReception, "Channel B VDL reception"),
+    (2, ChannelAVdlTransmission, "Channel A VDL transmission"),
+    (3, ChannelBVdlTransmission, "Channel B VDL transmission"),
+    (
+        4,
+        OwnInformationNotBroadcast,
+        "Own information not broadcast"
+    ),
+    (5, Reserved, "Reserved"),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    NavStatus,
+    (0, UnderWayUsingEngine, "Under way using engine"),
+    (1, AtAnchor, "At anchor"),
+    (2, NotUnderCommand, "Not under command"),
+    (3, RestrictedManeuverability, "Restricted maneuverability"),
+    (4, ConstrainedByHerDraught, "Constrained by her draught"),
+    (5, Moored, "Moored"),
+    (6, Aground, "Aground"),
+    (7, EngagedInFishing, "Engaged in Fishing"),
+    (8, UnderWaySailing, "Under way sailing"),
+    (
+        9,
+        HazardousMaterialHighSpeed,
+        "Hazardous material - High Speed"
+    ),
+    (
+        10,
+        HazardousMaterialWingInGround,
+        "Hazardous material - Wing in Ground"
+    ),
+    (
+        11,
+        PowerDrivenVesslTowingAstern,
+        "Power-driven vessl towing astern"
+    ),
+    (
+        12,
+        PowerDrivenVesslPushingAhead,
+        "Power-driven vessl pushing ahead or towing alongside"
+    ),
+    (14, AisSart, "AIS-SART"),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    AisSpecialManeuver,
+    (0, NotAvailable, "Not available"),
+    (
+        1,
+        NotEngagedInSpecialManeuver,
+        "Not engaged in special maneuver"
+    ),
+    (2, EngagedInSpecialManeuver, "Engaged in special maneuver"),
+    (3, Reserved, "Reserved"),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    DirectionReference,
+    (0, True, "True"),
+    (1, Magnetic, "Magnetic"),
+    (2, Error, "Error"),
+    CouldNotParse
+);
+
+define_nmea_enum!(
+    Gns,
+    (0, Gps, "GPS"),
+    (1, Glonass, "GLONASS"),
+    (2, GpsGlonass, "GPS+GLONASS"),
+    (3, GpsSbasWaas, "GPS+SBAS/WAAS"),
+    (4, GpsSbasWaasGlonass, "GPS+SBAS/WAAS+GLONASS"),
+    (5, Chayka, "Chayka"),
+    (6, Integrated, "integrated"),
+    (7, Surveyed, "surveyed"),
+    (8, Galileo, "Galileo"),
+    CouldNotParse
```

Comment:
`CouldNotParse` feels like it should be an error vs an actual field. If we want to represent an Unknown LookupField then I would suggest naming it `UnknowLookupField` or alike